src/components/views/auth/InteractiveAuthEntryComponents.js

```diff
@@ -650,7 +650,7 @@ export class SSOAuthEntry extends React.Component {
     };
 
     onConfirmClick = () => {
-        this.props.submitAuthDict({});
+        this.props.submitAuthDict(null);
```

Comment:
This should still be empty because this is submitting the auth, not doing the initial auth.